api/kyverno/v1/image_verification_types.go

```diff
@@ -63,6 +63,12 @@ type ImageVerification struct {
 	// Repository is an optional alternate OCI repository to use for image signatures that match this rule.
 	// If specified Repository will override the default OCI image repository configured for the installation.
 	Repository string `json:"repository,omitempty" yaml:"repository,omitempty"`
+
+	// DigestMutate is an optional field which handles the tag-to-digest mutation for the provided image paths.
+	// Defaults to true.
+	// +kubebuilder:default:=true
+	// +kubebuilder:validation:Optional
+	DigestMutate *bool `json:"digestMutate,omitempty" yaml:"digestMutate,omitempty"`
```

Comment:
Could we instead name this mutateDigest? The general convention for naming is verbNoun.

Reply:
addressed